src/components/transactions/index.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Waypoint from 'react-waypoint';
+import tableStyle from 'react-toolbox/lib/table/theme.css';
+import { transactions } from '../../utils/api/account';
+import TransactionsHeader from './transactionsHeader';
+import TransactionRow from './transactionRow';
+
+class Transactions extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      transactions: [],
+      offset: 0,
+      loadMore: true,
+      length: 1,
+    };
+  }
+
+  componentDidMount() {
+    this.loadMore();
+  }
+
+  loadMore() {
+    if (this.state.loadMore && this.state.length > this.state.offset) {
+      this.setState({ loadMore: false });
+      transactions(this.props.activePeer, this.props.address, 20, this.state.offset)
+      .then((res) => {
+        const list = res.transactions.map(transaction => (
+          <TransactionRow address={this.props.address}
+            key={transaction.id}
+            tableStyle={tableStyle}
+            value={transaction}>
+          </TransactionRow>
+        ));
+        this.setState({
+          transactions: this.state.transactions.concat(list),
```

Comment:
I think list of pending transactions should store in redux store and they come from voting component which hasn't been developed yet.So we can't implement them now.We can't store list of transactions in redux store because they come from server.